OsmAnd/src/net/osmand/plus/quickaction/actions/AddPOIAction.java

```diff
@@ -79,6 +80,29 @@ public AddPOIAction(QuickAction quickAction) {
 		super(quickAction);
 	}
 
+	@Override
+	public int getIconRes(Context context) {
+		if (context instanceof MapActivity) {
```

Comment:
Why do we need this check?

Reply:
Yes it's redundant. Context is always instance of MapActivity.